content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -250,3 +251,5 @@ By default truncate is disabled in both `webhook` and `log`, a cluster administr
 ## {{% heading "whatsnext" %}}
 
 * Learn about [Mutating webhook auditing annotations](/docs/reference/access-authn-authz/extensible-admission-controllers/#mutating-webhook-auditing-annotations).
+* Read the [reference for `audit.k8s.io` API group](/docs/reference/config-api/apiserver-audit.v1/).
```

Comment:
nit: we could actually link directly to the Event and Policy definitions.

Reply:
good idea, will do.

Reply:
good idea, will do.